lib/internal/process/stdio.js

```diff
@@ -2,7 +2,26 @@
 
 exports.getMainThreadStdio = getMainThreadStdio;
 
-function dummyDestroy(err, cb) { cb(err); }
+function dummyDestroy(err, cb) {
+  // SyncWriteStream does not use the stream
+  // destroy mechanism for some legacy reason.
+  // TODO(mcollina): remove when
+  // https://github.com/nodejs/node/pull/26690 lands.
```

Comment:
What exactly are you intending to remove? The `dummyDestroy()` function? It’s intentional that stdio streams, SyncWriteStream or not, cannot “really” be closed (so that file descriptors 0, 1, 2 don’t accidentally get closed).

Reply:
#26690 was merged so this needs to go now?

Reply:
We can! However that was a semver-major commit and this is not. I would prefer to issue a follow-up PR that will not be backported. This can (and should) be backported.